docs/source/standard/contract.rst

```diff
@@ -132,7 +132,7 @@ Schema
     The period during which tax concessions are made for some reason.
```

Comment:
the whole structure has to be within **_docs/source/standard/auction.rst_** (not _docs/source/standard/contract.rst_)